### ISSUE TITLE
Cant log in with connect button

### ISSUE THREAD

**Issue by HaxtorMoogle** [partkeepr/PartKeepr#511]
When I try to login by clicking the connect button I am told that my user name or password is wrong
To login I have to hit the return key after I enter in my password.
![bug](https://cloud.githubusercontent.com/assets/768681/11478452/18a480b6-975b-11e5-848f-266ac36ad451.png)

**Comment by felicitus** [comment]
I just tried to reproduce, but it works fine here.

**Comment by felicitus** [comment]
Is your installation reachable over the internet? If so, could you create an user for me so that I can try to reproduce it on your installation?

**Comment by Peter2016** [comment]
I have encountered the same problem. I thought I would set up Partkeepr 0.75 without touching my existing Partkeepr-0.1.9. I followed the same successful process that I have previously tried in creating additional independent databases using Partkeepr-0.1.9. I created a new database in mysql and a new folder for Partkeepr on my server. When I ran setup all was fine until it reached the username and password. It indicated that a previous installation had been detected and greyed out the username and password. I assumed that it had picked up the username and password from my other install somehow. Everything else went smoothly.
When I tried to login I got the "Username or password invalid" error for my old username and password. I also tried the admin/admin combination but got the same error.
